r/R/table.R

```diff
@@ -149,6 +149,77 @@ Table$create <- function(..., schema = NULL) {
 #' @export
 names.Table <- function(x) x$ColumnNames()
 
+#' @export
+rbind.Table <- function(...) {
```

Comment:
There is a ConcatenateTables C++ function in table.h. Any reason not to use it?

Reply:
No, I'll switch to using that. That also opens up the possibility of merging schemas since the existing function has an option for that. I think I'll also add a `concat_tables(..., unify_schemas = TRUE)` function, since I don't think we want that default behavior in rbind.

Reply:
Great idea